caluma/conftest.py

```diff
@@ -37,3 +39,41 @@ def assert_execution_result(self, result):
 
     with GraphQlSnapshotTest(request) as snapshot_test:
         yield snapshot_test
+
+
+@pytest.fixture
+def info(rf):
```

Comment:
What is this info fixture? Might need a docstring

Reply:
Added docstring.